src/main/java/frc/robot/Main.java

```diff
@@ -20,6 +23,13 @@ private Main() {}
    * <p>If you change your main robot class, change the parameter type.
    */
   public static void main(String... args) {
-    RobotBase.startRobot(Robot::new);
+    Random rand = new Random(); 
+    int upperbound = 10;
+    int int_random = rand.nextInt(upperbound); 
+
+    if (int_random <= 9){
+      RobotBase.startRobot(Robot::new);
+    }
```

Comment:
As entertaining as I find this, let's not actually commit this kind of change to a main branch.